flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -60,35 +67,48 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
     bool enableAutoTransactions = true,
     Duration autoFinishAfter = const Duration(seconds: 3),
     bool setRouteNameAsTransaction = false,
+    bool enableAppStartTracking = true,
```

Comment:
Does it mean that the App start on Flutter only works if you have set the `SentryNavigatorObserver`?
We should offer a public API to let people call it manually in case they are not using the `SentryNavigatorObserver`.
e.g. `SentryFlutter.setAppStartFinish(DateTime time = DateTimow.now()` or something similar.